src/components/Category.vue

```diff
@@ -1,9 +1,12 @@
 <template>
-  <div class="feedback-container__category">
-    <p class="feedback-container__category--label">{{ projectFeedback.rubric_category_name }} </p>
+  <div class="feedback-container__category s-content">
```

Comment:
I think the `s-content` here is referencing something from the Savile style sheet. Cool!